src/applications/mhv/secure-messaging/containers/LandingPageAuth.jsx

```diff
@@ -58,9 +59,16 @@ const LandingPageAuth = () => {
     updatePageTitle(PageTitles.DEFAULT_PAGE_TITLE_TAG);
   }, []);
 
+  useEffect(() => {
+    const h1 = document.querySelector('h1');
+    if (h1 && h1.textContent === 'Messages') {
+      setIsLandingPage(true);
+    }
+  }, []);
+

```

Comment:
I wonder if this is even needed? Is there a scenario where this component is rendered on a non-landing page?

Reply:
Your thought is correct, this prop is being used in the AlertBackgroundBox component to identify whether or not the user's current location is on the Messages screen. If the user is not on Messages then it handles an alternate alert message... I guess this can be moved?

Reply:
I think so too, we can safely remove this and just pass a prop